test/change_health/response/claim/report/report_835_service_line_test.rb

```diff
@@ -0,0 +1,124 @@
+require 'test_helper'
+
+class Report835DataTest < Minitest::Test
```

Comment:
Name should probably be Report835ServiceLineTest

Reply:
whoopsies!